main.py

```diff
@@ -0,0 +1,68 @@
+import streamlit as st
+from PIL import Image
+import random
+import pandas as pd
+
+
+#Functions
+def function_images():
+    cosine_distance = []
+    data = []
+
+    if option == 'Text query':
+        st.write(f'Output images and cosine distance for text query: {text}')
+    else:
+        st.write(f'Output images and cosine distance for current input image: {file.name}')
+
+    for i in range(values):
+        cosine_distance.append(random.random())
+
+    input_format = {str(f'({i+1})') + '.jpg': j for i, j in zip(range(values), cosine_distance)}

```

Comment:
Maybe should use`f'({i}).jpg'` and `range(1, values)`

Reply:
Thanks, yes, it can be more simplier